VictoriaAlbert/ImageExtension.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ImageExtension.swift
+//  VictoriaAlbert
+//
+//  Created by Edward Anchundia on 11/10/16.
+//  Copyright © 2016 Edward Anchundia. All rights reserved.
+//
+
+import UIKit
+
+let cacheImage = NSCache<NSString, UIImage>()
+extension UIImageView {
+    func downloadImage(urlString: String) {
```

Comment:
Let's move away from this extension and use the APIManager for image downloading.